src/app/rpmostree-dbus-helpers.c

```diff
@@ -1154,6 +1155,27 @@ get_modifiers_variant (const char   *set_refspec,
   vardict_insert_strv (&dict, "override-remove-packages", override_remove_pkgs);
   vardict_insert_strv (&dict, "override-reset-packages", override_reset_pkgs);
 
+  if (local_repo_remote)
+    {
+      /* Unfortunately, we can't pass an fd to a dir through D-Bus on el7 right now. So for
+       * now, we just pass the path. Once that's fixed (or we no longer care about
+       * supporting this feature on el7), we can go back to passing a handle. See:
+       * https://bugzilla.redhat.com/show_bug.cgi?id=1672404 */
```

Comment:
Hm; maybe we could make this build-time conditional on EL7?  And pass the handle on other platforms?

Reply:
Hmm yeah, I think that'd work nicely.